### PR TITLE
firstcommit

### PR DESCRIPTION
**Pull request by jayareena**

### CODE REVIEW

File: src/Pages/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import Invite from '../Component/Invite/Invite';
+import Trending from '../Component/Trending/Trending';
+import Dealoftheday from '../Component/Dealoftheday/Dealoftheday';
+import Shopbycategories from '../Component/Shopbycategories/Shopbycategories';
+import Banner from '../Component/Banner/Banner';
+import Trendingoffers from '../Component/Trendingoffers/Trendingoffers'
+import Testimonial from '../Component/Testimonial/Testimonial'
+import Blogs from '../Component/Blogs/Blogs';
+import Features from '../Component/Features/Features'
+import Aboutus from '../Component/Aboutus/Aboutus'
```
Comment (APJ-7) [inline src/Pages/Home.jsx:13]: Sort in alphabetical order

File: src/Pages/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      <Herosection />
+      <Trending />
+      <Dealoftheday />
+      <Banner />
+      <Trendingoffers />
+      <Shopbycategories />
+      <Testimonial />
+      <Blogs />
+      <Features />
+      <Aboutus />
```
Comment (APJ-7) [inline src/Pages/Home.jsx:29]: Use proper indentation

File: src/Pages/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      
+
```
Comment (APJ-7) [inline src/Pages/Home.jsx:31]: Remove white spaces and empty lines

File: src/Component/assets/data.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
+import p1_img from './Product1.png'
+import p2_img from './Product2.png'
+import p3_img from './Product3.png'
+import p4_img from './Product4.png'
+
+let data_product = [
```
Comment (APJ-7) [inline src/Component/assets/data.js:6]: use const insted of let.
use camelCase for variable naming.
https://dev.to/sathishskdev/part-1-naming-conventions-the-foundation-of-clean-code-51ng

File: src/Component/Testimonial/Testimonial.jsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import customer from "../assets/customer.png";
+import starIcon from '../assets/star_icon.png';
+
+const Testimonial = () => {
+  return (
+    <div className="flex flex-col items-center px-6 py-6 w-full h-auto md:w-[700px] md:h-[400px] mx-auto border border-gray-300 rounded-lg shadow-lg">
```
Comment (APJ-7) [inline src/Component/Testimonial/Testimonial.jsx:7]: move custom styles to tailwind css config file and use here.

**Comment by jayareena** [comment]
localhost_3000_ (1)

**Review by APJ-7 — CHANGES_REQUESTED** [review]

**Pull request merged**